apps/src/templates/manageStudents/ShowSecret.jsx

```diff
@@ -143,22 +149,14 @@ class ShowSecret extends Component {
                 style={styles.image}
               />
             )}
-            <span data-for={tooltipId} data-tip>
-              <Button
-                __useDeprecatedTag
-                onClick={this.reset}
-                color={Button.ButtonColor.blue}
-                text={i18n.reset()}
-                style={styles.reset}
-                disabled={resetDisabled}
```

Comment:
Setting this button to disabled if the user in the view is a teacher is also being removed, as this button will not be viewable anymore, if the user in this view is a teacher.